metron-interface/metron-config/src/app/sensors/reducers/sensors.reducers.spec.ts

```diff
@@ -534,7 +534,11 @@ describe('sensors: selectors', () => {
     expect(merged[1].status).toEqual(state.sensors.statuses.items[1]);
     expect(merged[2].status).toEqual(state.sensors.statuses.items[2]);
 
+<<<<<<< HEAD
```

Comment:
@ruffle1986 It looks like there is a merge conflict still needing to be resolved here.